packages/react-core/src/layouts/Level/Level.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly-next/layouts/Level/styles.css';
+
+const propTypes = {
+  hasGutter: PropTypes.bool,
+  className: PropTypes.string,
+  children: PropTypes.node
+};
+const defaultProps = {
+  hasGutter: false,
+  className: '',
+  children: null
+};
+
+const Level = ({ hasGutter, className, children, ...props }) => (
+  <div
+    {...props}
+    className={css(
+      styles.level,
+      hasGutter ? styles.modifiers.gutter : '',
```

Comment:
Nit pick and fine to leave as is: The `css` utility will take care of undefined or null values.  This can be written as `hasGutter && styles.modifiers.gutters`